pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -784,7 +786,7 @@ func GetAuditConfig(auditConfig configapi.AuditConfig) (audit.Backend, auditpoli
 		// backwards compatible writer to regular log
 		writer = cmdutil.NewGLogWriterV(0)
 	}
-	backend = auditlog.NewBackend(writer, auditlog.FormatLegacy)
+	backend = auditlog.NewBackend(writer, auditlog.FormatLegacy, auditv1alpha1.SchemeGroupVersion)
```

Comment:
why not the beta version?

Reply:
I don't want to do it in the rebase, I'll do it post, I have a trello card for it.